core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3105,4 +3281,182 @@ public Long getValue() {
 
     private Metrics() {} // prevent instantiation
   }
+
+  /**
+   * An audit context for file system master.
+   */
+  @NotThreadSafe
+  public final class MasterAuditContext implements AuditContext {
```

Comment:
I strongly suggest moving this class out as a separate java file. `DefaultFileSystemMaster` is too big. we always try best to keep it as thin as possible.